csharp/beetlex/config.yaml

```
@@ -0,0 +1,3 @@
+framework:
+  website: github.com/beetlex-io
```

```suggestion
  website: beetlex.io
```
or
```suggestion
  github: beetlex-io
```
in the first, the website will be backlinked, in the second it will be github

---

thanks